tests/services/test_spark.py

```diff
@@ -0,0 +1,13 @@
+# go three folders up
+import logging
+
+from ...app.services.spark_oceandb import FileLoader, SparkSession
+
+logging.info('Starting Spark Session!')

```

Comment:
All the tests should be within function wich starts from `test_*`
Here is a template for the test, please use it.

    class TestLoadExcelFile:
        def test():
            ...